app/components/TestBug.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Typography, Box, ButtonProps } from '@hero-design/react';
+
+import emptyBanner from './assets/empty-banner.png';
+
+console.log('test');
+
+const EmptyImg = styled.img`
+  height: 172px;
+  margin-bottom: ${({ theme }) => theme.space.small}px;
+`;
+
+interface EmptyTableViewProps {
+  message: string;
+  buttonText: ButtonProps['text'];
+  buttonIcon?: ButtonProps['icon'];
+  onButtonClick: ButtonProps['onClick'];
+}
+
+const EmptyTableView = ({
+  message,
+  buttonText,
+  buttonIcon = 'moneybag-outlined',
+  onButtonClick,
+}: EmptyTableViewProps) => (
+  <Box sx={{ textAlign: 'center' }}>
+    <EmptyImg src={emptyBanner} alt="Empty" />
+    <Typography.Text
+      fontSize={14}
+      sx={{ color: 'bodyText', mb: 'xxlarge', whiteSpace: 'pre-wrap' }}
+    >
+      {message}
+    </Typography.Text>
+
+    <Button
+      icon={buttonIcon}
+      onClick={onButtonClick}
+      sx={{
```

Comment:
Snowflake detected! A component is customized via sx prop. Make sure to not use [prohibited CSS properties](https://docs.google.com/spreadsheets/d/1Dj8vqLdFaf-CSaSVoYqyYZIkGqF6OoyP7K4G1_9L62U/edit?usp=sharing).